docker/package/tezos_voting_wizard.py

```diff
@@ -50,14 +50,15 @@
 welcome_text = """Tezos Voting Wizard
 
 Welcome, this wizard will help you to vote in the Tezos protocol amendment process.
-Please note that for this you need to already have set up the baking infrastructure,
-normally on mainnet, as only bakers can submit ballots or proposals.
+Please note that to vote on mainnet, you need to already have one of the following setups:
+- the baking infrastructure on mainnet;
+- a connected ledger device with Tezos Wallet app running;
```

Comment:
@pasqu4le already mentioned it as a comment to the commit message, but I'll repeat that we don't actually limit the ways secret keys are provided